src/running/dto/running.dto.ts

```diff
@@ -0,0 +1,15 @@
+import { IsArray, IsString } from 'class-validator';
+
+export type runData = {
+  latitude: number;
+
+  longitude: number;
```

Comment:
```suggestion
import { IsArray, IsNumber, IsString } from 'class-validator';

export type runData = {
  @IsNumber()
  latitude: number;

  @IsNumber()
  longitude: number;
```
여기에는 이렇게 IsNumber 안달아줘도 되나?
Suggest 기능이 있길래 한 번 써봤음 ㅋㅋㅋㅋㅋㅋ